src/main/java/org/mitre/synthea/export/FhirStu3.java

```diff
@@ -810,6 +820,313 @@ private static BundleEntryComponent encounterClaim(BundleEntryComponent personEn
+        allowedAmount.getCategory()
+            .getCoding()
+            .add(new Coding()
+                .setCode("https://bluebutton.cms.gov/resources/variables/line_alowd_chrg_amt")
+                .setSystem("https://bluebutton.cms.gov/resources/codesystem/adjudication")
+                .setDisplay("Line Allowed Charge Amount"));
+        allowedAmount.getAmount()
+            .setValue(item.getNet().getValue())
+            .setSystem("urn:iso:std:iso:4217")
+            .setCode("USD");
+
+        ExplanationOfBenefit.AdjudicationComponent indicatorCode =
+            new ExplanationOfBenefit.AdjudicationComponent();
+        indicatorCode.getCategory()
+            .getCoding()
+            .add(new Coding()
+                .setCode("https://bluebutton.cms.gov/resources/variables/line_prcsg_ind_cd")
+                .setSystem("https://bluebutton.cms.gov/resources/codesystem/adjudication")
+                .setDisplay("Line Processing Indicator Code"));
+        indicatorCode.getReason()
+            .addCoding()
+            .setCode("A")
+            .setSystem("https://bluebutton.cms.gov/resources/variables/line_prcsg_ind_cd")
+            .setDisplay("Allowed");
+
+
+        // assume deductible is 0
+        ExplanationOfBenefit.AdjudicationComponent deductibleAmount =
+            new ExplanationOfBenefit.AdjudicationComponent();
+        deductibleAmount.getCategory()
+            .getCoding()
+            .add(new Coding()
+                .setCode("https://bluebutton.cms.gov/resources/variables/line_bene_ptb_ddctbl_amt")
+                .setSystem("https://bluebutton.cms.gov/resources/codesystem/adjudication")
+                .setDisplay("Line Beneficiary Part B Deductible Amount"));
+        deductibleAmount.getAmount()
+            .setValue(0)
+            .setSystem("urn:iso:std:iso:4217")
+            .setCode("USD");
+
+        List<ExplanationOfBenefit.AdjudicationComponent> adjudicationComponents = new ArrayList<>();
+        adjudicationComponents.add(coinsuranceAmount);
+        adjudicationComponents.add(lineProviderAmount);
+        adjudicationComponents.add(submittedAmount);
+        adjudicationComponents.add(allowedAmount);
+        adjudicationComponents.add(deductibleAmount);
+        adjudicationComponents.add(indicatorCode);
+
+        itemComponent.setAdjudication(adjudicationComponents);
+        // the total payment is what the insurance ends up paying
+        totalPayment += 0.8 * item.getNet().getValue().doubleValue();
+      }
+      eobItem.add(itemComponent);
+    }
+
+    eob.setItem(eobItem);
+
+    Money payment = new Money();
+    payment.setValue(totalPayment)
+        .setCode("USD")
+        .setSystem("urn:iso:std:iso:4217");
+    eob.setPayment(new ExplanationOfBenefit.PaymentComponent()
+        .setAmount(payment));
+
+    // Hardcoded
+    List<Reference> recipientList = new ArrayList<>();
+    recipientList.add(new Reference("#2"));
```

Comment:
Having `#2` and `#3` here will not validate because there are no providers with those IDs.

I suggest you follow the Blue Button approach until we add individual clinicians with fake NPIs:

```json
"requester": {
                            "agent": {
                                "identifier": {
                                    "system": "http://hl7.org/fhir/sid/us-npi",
                                    "value": "999999999999"
                                }
                            }
                        },
                        "recipient": [
                            {
                                "identifier": {
                                    "system": "http://hl7.org/fhir/sid/us-npi",
                                    "value": "999999999999"
                                }
                            }
                        ]
```